build-system/tasks/lint.js

```diff
@@ -133,7 +149,20 @@ function lint() {
     options.fix = true;
   }
   if (argv.files) {
-    config.lintGlobs[config.lintGlobs.indexOf('**/*.js')] = argv.files;
+    config.lintGlobs =
+        config.lintGlobs.filter(e => e !== '**/*.js').concat(argv.files);
+  }
+  if (process.env.TRAVIS_PULL_REQUEST || process.env.LOCAL_PR_CHECK) {
+    config.lintGlobs =
+        config.lintGlobs.filter(e => e !== '**/*.js').concat(getLintFiles());
+    // Override .eslintrc settings here.
+    options['rules'] = {
+      // TODO(rsimha, #15255): This should error by default in .eslintrc.
+      'valid-jsdoc': 2,
```

Comment:
This removes the options we passed to the eslint configuration.

Reply:
Fixed.